src/main/java/dao/VolunteerServiceDAOStub.java

```diff
@@ -10,7 +10,8 @@
 @Repository
 public class VolunteerServiceDAOStub implements VolunteerServiceDAO {
 
-    List<Volunteer> allVolunteers = new ArrayList<Volunteer>();
+    //List<Volunteer> allVolunteers = new ArrayList<Volunteer>();
```

Comment:
Avoid committing commented-out code to version control, as version control offers better approaches:
1) If you are curious what used to be there, diff commits to see history.
2) If it is an experimental feature, use a feature branch to keep it separate from master.